packages/sidecar/src/cli/run.ts

```diff
@@ -0,0 +1,82 @@
+import { spawn } from "node:child_process";
+import { readFileSync } from "node:fs";
+import * as path from "node:path";
+import { logger } from "src/logger.js";
+import type { CLIHandlerOptions } from "src/types/cli.js";
+import { getSpotlightURL } from "src/utils/extras.js";
```

Comment:
these imports fail when running the compiled code, node thinks `src` is a module. we could use relative imports for this. 

<img width="1256" height="650" alt="Image" src="https://github.com/user-attachments/assets/4b537d5b-d643-47b1-af3d-f20ea5075c81" />